File: nagasena/src/org/openexi/scomp/GrammarCache4Grammar.java
package org.openexi.scomp;

import org.openexi.proc.common.GrammarOptions;
import org.openexi.proc.grammars.GrammarCache;
import org.openexi.schema.GrammarSchema;

final public class GrammarCache4Grammar {
  
  public static final GrammarCache m_grammarCache;
  static {
    m_grammarCache = new GrammarCache(GrammarSchema.getEXISchema(), GrammarOptions.STRICT_OPTIONS); 
  }
  
  public static final GrammarCache getGrammarCache() {
    return m_grammarCache;
  }
  
  private GrammarCache4Grammar() {
  }

}

File: nagasenacs/src/Nagasena/Scomp/GrammarCache4Grammar.cs
namespace Nagasena.Scomp {

  using GrammarOptions = Nagasena.Proc.Common.GrammarOptions;
  using GrammarCache = Nagasena.Proc.Grammars.GrammarCache;
  using GrammarSchema = Nagasena.Schema.GrammarSchema;

  /// <exclude/>
  public sealed class GrammarCache4Grammar {

    public static readonly GrammarCache m_grammarCache;
    static GrammarCache4Grammar() {
      m_grammarCache = new GrammarCache(GrammarSchema.EXISchema, GrammarOptions.STRICT_OPTIONS);
    }

    public static GrammarCache GrammarCache {
      get {
        return m_grammarCache;
      }
    }

    private GrammarCache4Grammar() {
    }

  }

}